package.json

```diff
@@ -21,7 +22,7 @@
       "require": "./dist/client/moduk-frontend.umd.js"
     },
     "./dist/*": "./dist/*",
-    "./src/css/*": "./src/css/*"
+    "./src/*": "./src/*"
```

Comment:
What's in `src/` is already controlled by `files` above, so this doesn't need to specify specific `src/` subdirectories.